Lombiq.Tests.UI/Lombiq.Tests.UI.csproj

```diff
@@ -79,4 +79,16 @@
     <ProjectReference Include="..\Lombiq.Tests.UI.Shortcuts\Lombiq.Tests.UI.Shortcuts.csproj" />
   </ItemGroup>
 
+  <Target Name="DownloadNpmPackages" Inputs="package.json" Outputs="node_modules\package.json" BeforeTargets="CollectPackageReferences">
```

Comment:
Great idea but please use [NPM Targets](https://github.com/Lombiq/NPM-Targets) for this instead.

Reply:
Changed to use `Lombiq.Npm.Targets`.